dot/rpc/modules/state_test.go

```diff
@@ -1129,3 +1129,147 @@ func TestStateModuleQueryStorage(t *testing.T) {
 		})
 	}
 }
+
+func TestStateModuleQueryStorageAt(t *testing.T) {
+	t.Parallel()
+	errTest := errors.New("test error")
+
+	type fields struct {
+		storageAPIBuilder func(ctrl *gomock.Controller) StorageAPI
+		blockAPIBuilder   func(ctrl *gomock.Controller) BlockAPI
+	}
+	type args struct {
+		in0 *http.Request
+		req *StateStorageQueryAtRequest
+	}
+	tests := map[string]struct {
+		fields    fields
+		args      args
+		errRegexp string
```

Comment:
no need for a regexp, the only error case is predictable (`getting value by block hash: test error`)